# Drone_stats.msg
# Accelerometer data
float32 accX
float32 accY
float32 accZ

# Gyroscope data
float32 gyroX
float32 gyroY
float32 gyroZ

# Magnetometer data
float32 magX
float32 magY
float32 magZ

# Orientation
float32 roll
float32 pitch
float32 yaw

# Altitude and battery data
float32 alt
float32 battery
float32 rssi

# Anchor data
float32 a1
float32 a2
float32 a3
